doc/sphinx/source/get-started/installation.rst

```diff
@@ -147,3 +153,36 @@ This will open a bash shell with the ``nnpdf`` environment already activated, wi
 all binaries and scripts from the NNPDF framework.
 
 .. _NNPDF GitHub Packages: https://github.com/NNPDF/nnpdf/pkgs/container/nnpdf
+
+
+.. _dependencies:
+
+Dependencies and requirements
+-----------------------------
+
+The NNPDF framework would not be possible with a number of dependencies.
+These are automatically installed when using conda and the full list can be consulted in the
+`conda recipe <https://github.com/NNPDF/nnpdf/blob/master/conda-recipe/meta.yaml#L20>`_ available in the repository.
+
+Below we list some of the most relevant external libraries than underpin the different aspects of this framework.
+
+::
+
```

Comment:
reportengine

Reply:
That's not really external so it is also part of the suggested citations. I'm also skipping EKO and pineappl for the same reasons.